source/frontend/src/features/mapSideBar/acquisition/add/AddAcquisitionForm.tsx

```diff
@@ -82,6 +83,8 @@ export const AddAcquisitionForm = React.forwardRef<
       innerRef={ref}
       initialValues={initialValues}
       validationSchema={validationSchema}
+      validateOnChange={false}
```

Comment:
What was the reason for this change?

Reply:
When changing the Type to 'Other' it's triggering the OtherDescription validation before even entering the value and/or submitting.